library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -83,21 +83,25 @@ static int pxToDp(float px) {
      * @param view view to animate.
      */
     public static void fadeOutAnim(final View view) {
-        ViewCompat.animate(view).cancel();
+        ViewCompat.animate(view).cancel
+                ();
         view.setAlpha(1F);
         view.setVisibility(VISIBLE);
         ViewCompat.animate(view)
                 .alpha(0F)
                 .withLayer()
                 .setDuration(SHORT_ANIM_TIME)
-                .setInterpolator(new AccelerateDecelerateInterpolator())
+                .setInterpolator( new AccelerateDecelerateInterpolator())
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L94

'(' is followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1905168930>*